src/utils/QCCalculator.cpp

```diff
@@ -396,29 +391,48 @@ class TOPPQCCalculator :
     at.colTypes.push_back("MS:1000285");
     UInt max = 0;
     Size below_10k = 0;
-    for (Size i = 0; i < exp.size(); ++i)
-    {
-      if (exp[i].getMSLevel() == 1)
+    std::vector<OpenMS::Chromatogram> chroms = exp.getChromatograms();
+    if (chroms.size() == 1) //real TIC from the mzML
```

Comment:
If exp.getChromatograms() results in an empty vector, then there is no chromatogram. 
If there are chromatogram(s) then there was a ```<chromatogram index="0" id="TIC" defaultArrayLength="29768">``` 
What is your point with a CV?